components/camel-coap/src/main/resources/META-INF/services/org/apache/camel/component/coaps

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
on master the META-INF is generated automatically during the component build, so you can safely remove this

Reply:
If I remove it then the build fails (mvn clean install in components/camel-coap)?

Reply:
you need to add the coaps scheme here:

https://github.com/apache/camel/pull/2891/files#diff-aa2b51fc80674de4afee8a0e0374a3e3R47

So it needs to look like:

@Component("coap,coaps")

Reply:
Thanks, fixed.